support-frontend/assets/pages/[countryGroupId]/components/thankyou.tsx

```diff
@@ -150,14 +150,15 @@ export function ThankYouComponent({
 				: order.paymentMethod;
 
 		successfulContributionConversion(
-			payment.originalAmount,
+			payment.finalAmount, // This is the final amount after discounts
```

Comment:
I'm not sure we should pass a temporary discounted amount to ? I think sending the full non-discounted `originalAmount` is preferable.

Reply:
Maybe we should ask Marketing what their preference is?

Reply:
Yes really was not sure, do you know who best to contact and will do that. Little uneasy as QM takes the original price but just did not seem right to me (for example the discounts on Tier3)

Reply:
I'll send you the Marketing contact regarding the Google Tag amount, I wouldn't worry about the QM amount here.

Reply:
Laetitia would like the price to include offer so will stick with final price. It does mis-align with QM but I have at least put a comment on both the QM and GoogleTagManager calls..